src/features/views/components/ViewColumnDialog/SurveyResponseConfig.tsx

```diff
@@ -46,6 +46,9 @@ const SurveyResponseConfig = ({
   const [surveyId, setSurveyId] = useState<number | null>();
   const [selectedQuestion, setSelectedQuestion] =
     useState<ZetkinSurveyQuestionElement | null>(null);
+  const [selectedColumnOption, setSelectedColumnOption] = useState<
+    SURVEY_QUESTION_OPTIONS | string
```

Comment:
I see that it's because `selectedColumnOption` can also contain the string representation of any option. It also seems that the ONE_OPTION is never used because instead the actual option is chosen.